docs/book/known_issues.md

```diff
@@ -0,0 +1,41 @@
+# vSphere CSI Driver - Known Issues
+
+This section lists the major known issues with VMware vSphere CSI driver. For complete list of issues please check our Github issues(https://github.com/kubernetes-sigs/vsphere-csi-driver/issues) page. If you notice an issue not listed in Github issues page, please do file a bug on the Github repo.
+
+- [Known Issues for Resize](#known_issues_for_resize)
```

Comment:
It would be good if you can remove this grouping and add ID for each known issue, so that during communication with customer we can share direct link to the issue.